caas/kubernetes/provider/storage.go

```diff
@@ -22,7 +22,7 @@ import (
 const (
 	// K8s_ProviderType defines the Juju storage type which can be used
 	// to provision storage on k8s models.
-	K8s_ProviderType = storage.ProviderType("kubernetes")
+	K8s_ProviderType = storage.ProviderType(CAASProviderType)
```

Comment:
this needs to stay as is because the storage type is different to the provider type even though the strings are the same value

Reply:
ic, reverted.